pkg/systemd/services/timer-dialog.jsx

```diff
@@ -106,45 +164,98 @@ const CreateTimerDialogBody = ({ owner }) => {
         />
     );
 
-    function onSubmit(event) {
-        setSubmitted(true);
+    const onSubmit = (event) => {
+        if (!event)
+            return false;
+
+        let failed = false;
+        let failMessage = _("Please complete the form");
 
-        if (event)
-            event.preventDefault();
+        event.preventDefault();
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18148-20230110-092821-56aa70de-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)